src/components/rewards-faq-modal/constants.ts

```diff
@@ -0,0 +1,27 @@
+export const rewardsFaq = [
+  {
+    content:
+      'A product does not exist without the people who use it. You — the user — are inherently valuable, as is the content you create, private or public. Big Tech social media recognizes that and exploits you for it, leaving you with nothing. At ZERO, we recognize the wealth of your content by rewarding you with $ZERO tokens. Your social interactions are valuable; you deserve to share that value.',
+    title: 'This seems too good to be true, how can I earn for doing nothing?',
+  },
+  {
+    content:
+      '$ZERO is a cryptocurrency token (ERC-20) that is built on the Ethereum (ETH) blockchain. It is the native currency of the ZERO ecosystem where it powers our suite of native zApps, including our powerful domain ownership platform, ZNS (ZERO Naming System). $ZERO can also be swapped for fiat currencies (like the USD-backed $USDC) at major DeFI exchanges, like Uniswap. As a Web3 token, $ZERO also represents voting power within the ZERO network. Holding $ZERO allows you to vote to shape the future of the ZERO Messenger and broader ecosystem; holding more $ZERO gives you more voting power. If you’re interested viewing an early application of ZERO technology in action, check out the Wilder World Metaverse and DAO.',
```

Comment:
@dalefukami I have added the original content. But here we have hyperlinks & line breaks in the text. Do you know how to add it? The `zUI` is only accepting the content part as "string"

[reference](https://www.figma.com/file/aETpyuG2AKjNcQtCjldcX0/zOS-Messenger-%26-Notifications?type=design&node-id=7563-326267&t=cp9mUEtbQ6TnEfWl-0) 
<img width="631" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/d90de937-d6b9-4d4f-a3bd-8759949f62b3">

Reply:
Ah, yeah, I guess the zUI component doesn't support that at the moment. We would have to modify it to take in actual jsx instead of just text.

Reply:
Oh, just looked. It seems like `content` CAN be jsx. So:
```
{
  title: 'blah',
  content: <div >lajlajflj <a href='...'>...</a>.....</div>
}
```
Should work?

Reply:
doesn't work. it is throwing a type error :(

Reply:
done